website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -95,6 +95,8 @@ The following arguments are supported:
 
 * `os_disk_size_gb` - (Optional) The Agent Operating System disk size in GB. Changing this forces a new resource to be created.
 
+* `os_disk_type` - (Optional) The Agent Operating System disk type. Can be `Managed` or `Ephemeral`. Defaults to `Managed`. Changing this forces a new resource to be created.
```

Comment:
AKS uses [the description for this field](https://docs.microsoft.com/en-us/rest/api/aks/agentpools/createorupdate) `OS disk type to be used for machines in a given agent pool.` for this - but I think we could reword that slightly so it's clearer, so could we make this:

```suggestion
* `os_disk_type` - (Optional) The type of disk which should be used for the Operating System. Possible values are `Ephemeral` and `Managed`. Defaults to `Managed`. Changing this forces a new resource to be created.
```